acrobot.py

```diff
@@ -10,18 +10,22 @@
 from pydrake.all import *
 from ilqr import IterativeLinearQuadraticRegulator
 import time
+import utils_derivs_interpolation
 
 ####################################
 # Parameters
 ####################################
 
-T = 1.5        # total simulation time (S)
-dt = 1e-2      # simulation timestep
+T = 10       # total simulation time (S)
```

Comment:
This time horizon seems a bit long. When I ran it the acrobot just sort of hung out for a while before finally swinging up